oxo.py

```diff
@@ -1,26 +1,130 @@
 class OxoBoard:
     def __init__(self):
         """ The initialiser. Initialise any fields you need here. """
-        raise NotImplementedError("TODO: implement __init__")
+        self.number_to_win_for = 3
+        self.rows = 3
+        self.cols = 3
+
+        self.numberOfTiles = self.rows * self.cols
+
+        # Create and initialise the game board
+        self.board = []
+        for row in range(self.rows):
+            self.board += [[0] * self.cols]
```

Comment:
Would be slightly more efficient to do `self.board.append([0] * self.cols)` -- here you're creating a temporary single-element list